### PR TITLE
OpenLineage LFAI&Data proposal

### PR DESCRIPTION
**Pull request by julienledem**

### CODE REVIEW

**Review by wslulciuc — APPROVED** [review]
🎉

File: proposal/OpenLineage.adoc [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+https://github.com/OpenLineage/OpenLineage
+
+*Statement on alignment with LF AI’s mission*:
+ 
+Lineage is a critical component in governing AI and Data projects.
+
+*Possible collaboration opportunities with current LF AI hosted projects*: (https://lfai.foundation/projects/)
+
+This standard is being developed in collaboration between the Marquez and Egeria projects.
+Additionally the Amundsen projecy is a candidate for implementing ingestion of OpenLineage events.
```
Comment (wslulciuc) [inline proposal/OpenLineage.adoc:23]: _Minor: _projecy_ -> _project_

Comment (julienledem) [reply proposal/OpenLineage.adoc:23]: thx

**Review by mobuchowski — APPROVED** [review]

**Review by collado-mike — APPROVED** [review]

**Comment by julienledem** [comment]
FYI: https://github.com/lfai/proposing-projects/pull/45

**Comment by julienledem** [comment]
This is now in the lfai repo: https://github.com/lfai/proposing-projects/pull/45

**Pull request closed without merging**